lib/packwerk/detect_stale_deprecated_references.rb

```diff
@@ -0,0 +1,14 @@
+# typed: true
+# frozen_string_literal: true
+
+require "packwerk/cache_deprecated_references"
+
+module Packwerk
+  class DetectStaleDeprecatedReferences < CacheDeprecatedReferences
+    def stale_violations?
+      @deprecated_references.any? do |_, deprecated_references_file|
```

Comment:
You could do…

```suggestion
      @deprecated_references.values.any? do |deprecated_references_file|
```

…instead if you prefer that.